skr-webhook/resources.yaml

```diff
@@ -98,23 +96,6 @@ spec:
           ports:
             - containerPort: 8443
               name: watcher-port
-        - name: request-sidecar
-          image: europe-docker.pkg.dev/kyma-project/prod/external/alpine:3.17.4
-          command:
-            - /bin/sh
-            - "-c"
-            - "sleep 60m"
-          imagePullPolicy: Always
-          volumeMounts:
```

Comment:
You can also remove the volumeMount for the tmp path for the server container and the emptyDir volume itself

Reply:
good catch 👍